src/funnels/Depositor.sol

```diff
@@ -0,0 +1,424 @@
+// SPDX-FileCopyrightText: © 2020 Lev Livnev <lev@liv.nev.org.uk>
+// SPDX-FileCopyrightText: © 2021 Dai Foundation <www.daifoundation.org>
```

Comment:
```suggestion
// SPDX-FileCopyrightText: © 2023 Dai Foundation <www.daifoundation.org>
```

Reply:
Implemented in https://github.com/makerdao/dss-allocator/pull/15